setup.cfg

```diff
@@ -23,8 +23,12 @@ url = https://pyaro.readthedocs.io
 [options]
 python_version = >=3.10
 install_requires =
-    numpy >= 1.13
+    numpy >= 1.13, <2.0.0
     importlib-metadata >= 3.6; python_version < "3.10"
+    xarray
+    netcdf4
+    cf-units
+    cftime
```

Comment:
xarray,netcdf4,cf-units and cftime are pretty big dependencies for pyaro, and are just needed for the relatltitude test.
Can you make the dependencies optional (and for testing)? Similar to the pandas dependency. As interface, pyaro should have hardly any dependencies.